js/RTCStatsReport.ts

```diff
@@ -1,20 +1,32 @@
-/**
- * Expose the RTCStatsReport class.
- */
-module.exports = RTCStatsReport;
+import { RTCStatsShim } from './RTCStats';
 
-function RTCStatsReport(data) {
-	data = data || {};
+export class RTCStatsReportShim implements RTCStatsReport {
```

Comment:
The original implementation of `RTCStatsReport` and `RTCStatsResponse` were inconsistent with the spec.  They had it so that a "Response" had multiple "Report"s.  Based on the spec (https://w3c.github.io/webrtc-pc/#rtcstatsreport-object), there should be one `RTCStatsResponse` with multiple `RTCStats`.   Also, the `RTCStats` data was nested in a `.data` member, but in chrome the data is all available at the top level.  I've rearranged the classes and data to match the spec and observed behavior in chrome, and kept the `.data` member for backward compatibility.

IOSRTC after change: 
![image](https://user-images.githubusercontent.com/3026298/98388569-2bb85b80-2010-11eb-9834-4addaeb5517f.png)

Chrome (for comparison)
![image](https://user-images.githubusercontent.com/3026298/98388660-468ad000-2010-11eb-9fe4-ccaa3e5065fa.png)

Reply:
Ok for me